sigs.yaml

```diff
@@ -1644,16 +1644,17 @@ sigs:
     - name: kubernetes-scalability-test-frameworks
       description: "[Described below](#kubernetes-scalability-test-frameworks)"
       owners:
-      - https://raw.githubusercontent.com/kubernetes/kubernetes/master/cluster/images/kubemark/OWNERS
-      - https://raw.githubusercontent.com/kubernetes/kubernetes/master/cmd/kubemark/OWNERS
-      - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/kubemark/OWNERS
-      - https://raw.githubusercontent.com/kubernetes/kubernetes/master/test/kubemark/OWNERS
-      - https://raw.githubusercontent.com/kubernetes/perf-tests/blob/master/clusterloader2/OWNERS
-      - https://raw.githubusercontent.com/kubernetes/perf-tests/blob/master/OWNERS
+      - https://github.com/kubernetes/kubernetes/blob/master/cluster/images/kubemark/OWNERS
```

Comment:
Why are these links changing? They should be the direct raw links.

Reply:
+1 - please revert everywhere

Reply:
Done.

Reply:
Also fixed some wrong links from before - PTAL.